src/windows/tile_inspector.c

```diff
@@ -570,6 +578,10 @@ void window_tile_inspector_open() {
 	window_tile_inspector_auto_set_buttons(window);
 }
 
+void window_tile_inspector_clear_clipboard() {
```

Comment:
Opening braces of functions should go onto their own line. (I'm leaving this comment once, but it applies to every instance in this PR, of course.)